packages/checksum/README.md

```diff
@@ -17,37 +17,83 @@ yarn add @iota/checksum
 
 ## API Reference
 
-    <a name="module_checksum..addChecksum"></a>
+    <a name="module_checksum.addChecksum"></a>
```

Comment:
![image](https://user-images.githubusercontent.com/19519564/84061746-7dd75100-a98c-11ea-87a2-e11bf465b447.png)

I think the indentation is making this render as a code snippet